crates/net/network/src/transactions/fetcher.rs

```diff
@@ -367,11 +375,15 @@ impl TransactionFetcher {
         metrics_increment_egress_peer_channel_full: impl FnOnce(),
     ) -> Option<Vec<TxHash>> {
         let peer_id: PeerId = peer.request_tx.peer_id;
+        let msg_version = new_announced_hashes.first().map(|hash| {
```

Comment:
looks like we're only using this in traces, 
tbh I don't know how they expand, but perhaps we should make this a closure?